stock_reserve/view/product.xml

```diff
@@ -1,25 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
 <openerp>
     <data noupdate="0">
-
-        <record model="ir.ui.view" id="product_template_form_view_reservation_button">
-            <field name="name">product.template.reservation.button</field>
-            <field name="model">product.template</field>
-            <field name="inherit_id" ref="product.product_template_only_form_view"/>
-            <field name="arch" type="xml">
-                <xpath expr="//div[@name='buttons']" position="inside">
-                    <button class="oe_inline oe_stat_button" name="action_view_reservations"
-                        type="object" groups="base.group_sale_salesman" icon="fa-lock">
-                        <field string="Stock Reservations" name="reservation_count" widget="statinfo" />
-                    </button>
-                </xpath>
-            </field>
-        </record>
-
```

Comment:
As this stanza was deleted, does this mean that you can not inspect the number of reserved product variants?